blobmodel/stochasticality.py

```diff
@@ -33,11 +39,13 @@ class DefaultBlobFactory(BlobFactory):
     def __init__(
         self,
         A_dist: str = "exp",
-        W_dist: str = "exp",
+        wx_dist: str = "deg",
+        wy_dist: str = "deg",
```

Comment:
Since you introduce `wy_dist`, do you also plan to implement different shapes for blobs in y?

Reply:
Yeah, that is already implemented